Objects/Converters/ConverterAutocadCivil/ConverterAutocadCivilShared/Converter.AutocadCivil.Utils.cs

```diff
@@ -126,14 +122,31 @@ public static void AddNameAndDescriptionProperty(string name, string description
   public static bool GetHandle(string str, out Handle handle)
   {
     handle = new Handle();
+    if (string.IsNullOrEmpty(str))
+    {
+      return false;
+    }
+
+    long l;
     try
     {
-      handle = new Handle(Convert.ToInt64(str, 16));
+      l = Convert.ToInt64(str, 16);
     }
-    catch
+    catch (ArgumentException)
     {
       return false;
     }
+    catch (FormatException)
+    {
+      return false;
+    }
+    catch (OverflowException)
+    {
+      return false;
+    }
```

Comment:
No change needed, but we've been discussing doing something like:

```csharp
catch (Exception ex) when (ex is ArgumentException or FormatException or OverflowException) {...}
```
When the handling of all cases is in essence the same, but we haven't settled on a preference yet. Just FYI this can be done.